starter_code/routes/index.js

```diff
@@ -1,9 +1,25 @@
 const express = require('express');
 const router  = express.Router();
+const Movie = require('../models/Movie');
 
+/* GET home page */
 /* GET home page */
 router.get('/', (req, res, next) => {
   res.render('index');
 });
 
+router.get('/movies', async (req, res, next) => {
+  const movies = await Movie.find()
+  res.render('movies', {movies});
+});
+
+router.get('/movies/:id', async (req, res, next) => {
+  const {id} = req.params
+  const movie = await Movie.findById(id)
+  res.render('movies-details', movie);
```

Comment:
It won't render the details of each movie because the file is called `movie-details`, not `movies-details`